accord-core/src/test/java/accord/impl/basic/RandomDelayQueue.java

```diff
@@ -63,6 +64,22 @@ public int compareTo(Item that)
             return c;
         }
 
+        @Override
+        public boolean equals(Object o)
+        {
+            if (o instanceof Pending) return item.equals(o);
+            if (this == o) return true;
+            if (o == null || getClass() != o.getClass()) return false;
+            Item item1 = (Item) o;
+            return time == item1.time && seq == item1.seq && item.equals(item1.item);
+        }
+
+        @Override
+        public int hashCode()
+        {
+            return Objects.hash(time, seq, item);
```

Comment:
We don't put this in a hash map I don't think, so let's not implement this.

Reply:
switched to `UnsupportedOperationException` and reran the tests